pennylane/devices/default_qutrit_mixed.py

```diff
@@ -165,21 +173,44 @@ def __init__(
         super().__init__(wires=wires, shots=shots)
         seed = np.random.randint(0, high=10000000) if seed == "global" else seed
         if qml.math.get_interface(seed) == "jax":
+            self._prng_seed = seed
             self._prng_key = seed
             self._rng = np.random.default_rng(None)
         else:
+            self._prng_seed = None
```

Comment:
As I said in my other comment, since we're making the PRNGKey changes in the future, we can get rid of these for now.

```suggestion
            self._prng_key = seed
            self._rng = np.random.default_rng(None)
        else:
```